### DISCUSSION TITLE
Why cant i get a float number returned when running python in Atom

### DISCUSSION THREAD

**Discussion by tommitchellflys** [atom/atom#22960] [Q&A]
HEY GUYS,
total newbie doing my first programming course,
we are using python3 i type a simple print(1/3) and can only get an integer returned , not a float number as i would in the python shell ? any ideas ?

code is litterally 

print(1/3) returns 3 
no decimal place in any numeric returns in ATOM 
any help would be awesome !

**Comment by icecream17** [comment]
What do you do to run python in atom?

**Reply by tommitchellflys** [reply]
i installed a package "ide-python" and "python terminal"
<img width="382" alt="Screenshot 2021-09-09 at 13 24 54" src="https://user-images.githubusercontent.com/81638562/132711700-eafd2c52-3cc1-4d69-a895-bd74642ac589.png">

**Reply by tommitchellflys** [reply]
as i said , complete noob just googling shit to try get it working and cant find anything about it i have probably kooked the setup 😩 haha , any help is appreciated !
